iidm/iidm-xml-converter/src/main/resources/xsd/referenceTerminals.xsd

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2023, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+-->
+<xs:schema version="1.0"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:iidm="http://www.powsybl.org/schema/iidm/1_11"
+           targetNamespace="http://www.powsybl.org/schema/iidm/ext/reference_terminals/1_0"
+           elementFormDefault="qualified">
+    <xs:import namespace="http://www.powsybl.org/schema/iidm/1_11" schemaLocation="iidm_V1_11.xsd"/>
+    <xs:element name="referenceTerminals">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="referenceTerminal"  minOccurs="0" maxOccurs="unbounded">
```

Comment:
You could have used the TerminalRef type of iidm.xsd like for slack terminal BUT:
 - maybe it is better to duplicate the type like this and avoid a dependency on base iidm version. we had quite a bad experience with that for slack terminal
 - was it what you have in mind?

Reply:
actually I had in mind to use iidm:TerminalRef ... pushed a commit in this sense.
What was the bad experience ? If you think it is better to duplicate, we can revert eb479531a6d58ef067adfd28d6847118dd5cfa35

Reply:
We have release multiple versions of slack terminal extension schema just to have up to date with last iidm one.
![image](https://github.com/powsybl/powsybl-core/assets/632675/e289c7c1-d5bd-435f-9240-cd791f60bb50)
We stopped since iidm 1.8 for an unknown reason.
I'm not sure what is the best practice in that case:
  - is it a problem to have an extension schema using a old iidm schema?
  - what could be the consequence?